tests/unit/cli/test_utils.py

```diff
@@ -41,3 +42,17 @@ def test_parse_unknown_options_returns_a_dict_of_known_options() -> None:
     assert isinstance(parsed_sample_args, dict)
     assert len(parsed_sample_args.values()) == 3
     assert parsed_sample_args["best_cat"] == '"aria"'
+
+
+def test_stack_config_has_right_contents_for_printing() -> None:
+    """Check that the stack config has the right components for printing"""
+    repo = Repository()
```

Comment:
Is this how we access the repository now in our tests? I am for sure outdated on the details but I thought we used to do this via pytest fixtures?

Reply:
Checked and `zenml/tests/unit/test_repository.py` and others like `zenml/tests/unit/orchestrators/test_context_utils.py` are using `Repository()` so I assume that's what we're doing?